lib/src/sql/statements/live.rs

```diff
@@ -69,6 +69,29 @@ impl LiveStatement {
 		trace!("Evaluated live query auth to {:?}", self_override.auth);
 		// Claim transaction
 		let mut run = txn.lock().await;
+		// Replace Params for condition
+		if let Some(condition) = self_override.cond {
+			let parsed = condition.compute(ctx, opt, txn, None).await?;
+			trace!("Evaluated live query condition {:?} to {:?}", condition, parsed);
```

Comment:
Not sure how to resolve this so that it doesn't evaluate to "false" but a non-parameterised condition
```
2023-09-06T18:13:10.423541Z TRACE rpc/call:process:executor: surrealdb::sql::statements::live: Evaluated live query condition Cond(Expression(Binary
 { l: Idiom(Idiom([Field(Ident("number"))])), o: Equal, r: Param(Param(Ident("number"))) })) to Bool(false) otel.kind="server" rpc.service="surreald
b" rpc.system="jsonrpc" rpc.jsonrpc.version="2.0" ws.id=153807b4-5eee-4ffa-9695-e5665270c24c rpc.method="query" otel.name="surrealdb.rpc/query" rpc.
jsonrpc.request_id="3"
```